tests/storage/iso/test_cifs_iso_sr_reboot.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import os
+from lib.common import wait_for
+from conftest import copy_tools_iso_to_iso_sr, check_iso_mount_and_read_from_vm, remove_iso_from_sr
+
+# Requirements:
+# - one XCP-ng host with a SAMBA/CIFS share for the SR and a VM
+
+@pytest.mark.usefixtures("cifs_iso_sr")
+class TestCIFSISOSR():
+    """
+    This test is longer because the host reboots between the creation of the SR and the test with the ISO file.
+    """
+
+    def test_iso_cifs(self, host, cifs_iso_sr, imported_vm):
```

Comment:
```suggestion
    def test_iso_mount_and_read_after_reboot(self, host, cifs_iso_sr, imported_vm):
```

Same comment for other "reboot" test files.